_tools/types/main.go

```diff
@@ -0,0 +1,340 @@
+// Generates a report about every driver wich UASTv2 types does it use,
+// both in actual test fixtures and though the source code.
+package main
+
+import (
+	"bufio"
+	"context"
+	"flag"
+	"fmt"
+	"log"
+	"net/http"
+	_ "net/http/pprof"
```

Comment:
Please consider grouping "blank" (side-effecting) imports separately from the others.